src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  --border-color: #e1e7ed;
+  --placeholder-color: #3d4e61;
+  --shadow-color: 0 2px 4px rgba(0, 0, 0, 0.05);
+  --hover-color: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+  --gradient-color: linear-gradient(#fff, #f6f6f7);
+  --background-image: url(images/Search.svg) no-repeat;
+  --imgsmall-size: 11px;
+  --font-family: "Avenir", sans-serif;
+}
+
+.search {
+  margin-top: 20px;
+}
+
+.input {
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  box-shadow: var(--shadow-color);
+  width: 100%;
+  background: var(--background-image) 26px 23px;
+  background-size: 19px;
+  padding: 27px 62px 28px;
```

Comment:
Do not hardcode paddings set the height of the form as it on the design, and then change the label display to block and set height 100%, same for input, but you will still need padding-left, and then double-check the top/left/width/height of background and also try to add box-sizing: border-box to `*` selector.